interface/billing/sl_eob_search.php

```diff
@@ -462,6 +462,82 @@ function npopup(pid) {
  return false;
 }
 
+function computeTableColumnTotal(tableId, colNumber)
+{
+  // find the table with id attribute tableId
+  // return the total of the numerical elements in column colNumber
+  // skip the top row (headers) and bottom row (where the total will go)
+		
+  var result = 0;
+		
+  try
+  {
+    var tableElem = window.document.getElementById(tableId); 		   
+    var tableBody = tableElem.getElementsByTagName("tbody").item(0);
+    var i;
+    var howManyRows = tableBody.rows.length;
+    for (i=1; i<(howManyRows-1); i++) // skip first and last row (hence i=1, and howManyRows-1)
+    {
+       var thisTrElem = tableBody.rows[i];
+       var thisTdElem = thisTrElem.cells[colNumber];			
+       var thisTextNode = thisTdElem.childNodes.item(0);
+       if (debugScript)
+       {
+          window.alert("text is " + thisTextNode.data);
+       } // end if
+
+       // try to convert text to numeric
+       var thisNumber = parseFloat(thisTextNode.data);
+       // if you didn't get back the value NaN (i.e. not a number), add into result
+       if (!isNaN(thisNumber))
+         result += thisNumber;
+	 } // end for
+		 
+  } // end try
+  catch (ex)
+  {
+     //window.alert("Exception in function computeTableColumnTotal()\n" + ex);
+     result = 0;
+  }
+  finally
```

Comment:
This finally is not for PHP to my knowledge. It is in the javascript section. The javascript will execute with or without the "finally" at the end. The finally statement is there to execute regardless of the result of the preceding code.